Makefile

```diff
@@ -14,27 +14,27 @@ PLATFORMS ?= linux_amd64 linux_arm64
 # ====================================================================================
 # Setup Go
 
-GO_REQUIRED_VERSION = 1.19
+#GO_REQUIRED_VERSION = 1.21
```

Comment:
Any particular reason for not removing these commented out versions?

Reply:
it was for testing reasons, fixed